doc/glossary.rst

```diff
@@ -433,6 +433,31 @@ General Concepts
     hyper-parameter
         See :term:`parameter`.
 
+    impute
+    imputation
+        Most machine learning algorithms require that their inputs have no
+        :term:`missing values`, and will not work if this requirement is
```

Comment:
Please add a backlink to the :term:`imputation` entry from the :term:`missing values` entry.

Reply:
Done.